pkg/util/file_windows.go

```diff
@@ -11,3 +14,21 @@ func SetFileModeForPath(name string, mode os.FileMode) error {
 func SetFileModeForFile(file *os.File, mode os.FileMode) error {
 	return nil
 }
+
+// ReadFile reads from a file
+func ReadFile(path string) (string, error) {
```

Comment:
Same comment as in the linux version

Reply:
I don't understand why this is OS-specific, there doesn't appear to be anything in either function that would differ across platforms.

Reply:
We could remove it in another PR (just in case we need to revert) and see if things are still working